core/domain/takeout_service_test.py

```diff
@@ -365,18 +371,18 @@ def set_up_non_trivial(self):
         5) Creates an ExplorationUserDataModel.
         6) Simulates completion of some activities.
         7) Simulates incomplete status of some activities.
-        8) Populates ExpUserLastPlaythroughModel of user.
-        9) Creates user LearnerPlaylsts.
-        10) Simulates collection progress of user.
-        11) Simulates story progress of user.
-        12) Creates new collection rights.
-        13) Simulates a general suggestion.
-        14) Creates new exploration rights.
-        15) Populates user settings.
-        16) Creates two reply-to ids for feedback.
-        17) Creates a task closed by the user.
-        18) Simulates user_1 scrubbing a report.
-        19) Creates new BlogPostModel and BlogPostRightsModel.
+        9) Populates ExpUserLastPlaythroughModel of user.
```

Comment:
Ditto here--(8) is now missing. Did you mean to add another line for the new model?

Reply:
Done. Yeah, I added a new line here for a new model. PTAL!